packages/vm/src/evm/interpreter.ts

```diff
@@ -21,8 +21,7 @@ export interface RunState {
   stack: Stack
   returnStack: Stack
   code: Buffer
-  validJumps: number[]
-  validJumpSubs: number[]
+  validJumps: Uint8Array
```

Comment:
Can we add a comment here what this Uint8Array means? It should describe that opcode at pc `index` has value 0 (default), 1 (jumpdest), 2 (beginsub)

Reply:
Does the comment I added align with your thoughts?

Reply:
Yep 👍 😄